packages/core/src/connectors/alipay/index.ts

```diff
@@ -0,0 +1,243 @@
+): Record<string, string> => {
+  const { biz_content: bizContent, privateKey, ...signParameters } = parameters;
+
+  if (bizContent) {
+    // eslint-disable-next-line @silverhand/fp/no-mutation
+    signParameters.biz_content = JSON.stringify(snakeCaseKeys(JSON.parse(bizContent)));
+  }
+
+  const decamelizeParameters = snakeCaseKeys(signParameters);
+
+  // eslint-disable-next-line @silverhand/fp/no-mutating-methods
+  const sortedParametersAsString = Object.keys(decamelizeParameters)
+    .sort()
+    .map((key) => {
+      const data = decamelizeParameters[key];
+
+      // Supported Encodings can be found at https://github.com/ashtuchkin/iconv-lite/wiki/Supported-Encodings
+      // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+      return data ? `${key}=${iconv.encode(data, signParameters.charset ?? 'UTF8')}` : '';
+    })
+    .filter((keyValueString) => keyValueString)
+    .join('&');
+
+  const sign = crypto
+    .createSign(alipaySigningAlgorithmMapping[signParameters.signType])
+    .update(sortedParametersAsString, 'utf8')
+    .sign(privateKey, 'base64');
+
+  return { ...decamelizeParameters, sign };
+};
+
+export const getAuthorizationUri: GetAuthorizationUri = async (redirectUri, state) => {
+  const { appId: app_id } = await getConnectorConfig<AlipayConfig>(metadata.id);
+
+  return `${authorizationEndpoint}?${stringify({
+    app_id,
+    redirect_uri: encodeURI(redirectUri), // The variable `redirectUri` should match {appId, appSecret}
+    scope,
+    state,
+  })}`;
+};
+
+export const getAccessToken: GetAccessToken = async (authCode) => {
+  // `error_response` and `alipay_system_oauth_token_response` are mutually exclusive.
+  type AccessTokenResponse = {
+    error_response?: {
+      code: string;
+      msg: string; // To know `code` and `msg` details, see: https://opendocs.alipay.com/common/02km9f
+      sub_code?: string;
+      sub_msg?: string;
+    };
+    sign: string; // To know `sign` details, see: https://opendocs.alipay.com/common/02kf5q
+    alipay_system_oauth_token_response?: {
+      user_id: string; // Unique Alipay ID, 16 digits starts with '2088'
+      access_token: string;
+      expires_in: string; // In seconds
+      refresh_token: string;
+      re_expires_in: string; // Expiring time of refresh token, in seconds
+    };
+  };
+
+  const config = await getConnectorConfig<AlipayConfig>(metadata.id);
+  const initSearchParameters = {
+    method: methodForAccessToken,
+    format: 'JSON',
+    timestamp: getFormattedDate(),
+    version: '1.0',
+    grant_type: 'authorization_code',
+    code: authCode,
+    charset: 'UTF8',
+    ...config,
+  };
+  const signedSearchParameters = signingPamameters(initSearchParameters);
+
+  const response = await got
+    .post(alipayEndpoint, {
+      searchParams: signedSearchParameters,
+      timeout: await getConnectorRequestTimeout(),
+    })
+    .json<AccessTokenResponse>();
```

Comment:
will got throw error?

Reply:
per tests, the server will always return code 200, and error codes or messages are wrapped in the AccessTokenResponse.error_response if errors are encountered.